cli/pcluster/config/validators.py

```diff
@@ -1066,3 +1072,59 @@ def _get_efa_enabled_instance_types(errors):
         )
 
     return instance_types
+
+
+def fsx_lustre_backup_validator(param_key, param_value, pcluster_config):
+    errors = []
+    warnings = []
+
+    try:
+        backups = boto3.client("fsx").describe_backups(BackupIds=[param_value]).get("Backups")
+        if backups is None or len(backups) <= 0:
+            errors.append("Backup '{0}' does not exist.".format(param_value))
+            return errors, warnings
+
+        backup = backups[0]
+        fsx_section = pcluster_config.get_section("fsx")
+        backup_deployment_type = backup.get("FileSystem").get("LustreConfiguration").get("DeploymentType")
+        deployment_type = fsx_section.get_param_value("deployment_type")
+        if deployment_type is not None:
+            errors.append(
+                FSX_MESSAGES["errors"]["unsupported_backup_param"].format(
+                    name="DeploymentType", value=backup_deployment_type
+                )
+            )
+
+        per_unit_storage_throughput = fsx_section.get_param_value("per_unit_storage_throughput")
+        backup_per_unit_storage_throughput = (
+            backup.get("FileSystem").get("LustreConfiguration").get("PerUnitStorageThroughput")
+        )
+        if per_unit_storage_throughput is not None:
+            errors.append(
+                FSX_MESSAGES["errors"]["unsupported_backup_param"].format(
+                    name="PerUnitStorageThroughput", value=backup_per_unit_storage_throughput,
+                )
+            )
+
+        storage_capacity = fsx_section.get_param_value("storage_capacity")
+        backup_storage_capacity = backup.get("FileSystem").get("StorageCapacity")
+        if storage_capacity is not None:
+            errors.append(
+                FSX_MESSAGES["errors"]["unsupported_backup_param"].format(
+                    name="StorageCapacity", value=backup_storage_capacity
+                )
+            )
+
+        if fsx_section.get_param_value("import_path"):
+            errors.append("When restoring file system from backup, import_path cannot be specified.")
+
+        if fsx_section.get_param_value("export_path"):
+            errors.append("When restoring file system from backup, export_path cannot be specified.")
+
+        if fsx_section.get_param_value("imported_file_chunk_size"):
+            errors.append("When restoring file system from backup, imported_file_chunk_size cannot be specified.")
+
```

Comment:
Same for these checks: since the error message is essentially the same for all of them, can you put them in a list and loop through it to make sure each isn't set?

Reply:
Instead of saying `restoring file system from backup` can you say `restoring an FSx Lustre file system from backup`?